# Historical relevance strength.
# Values between 0 (no relevance) and 1 (max relevance).
float32 value

# date of the last applied update for "value" field.
ltm/Date last_update

# next scheduled update for "value" field.
ltm/Date next_update
